gwpy/signal/qtransform.py

```diff
@@ -27,6 +27,7 @@
 
 import warnings
 from math import (log, ceil, pi, isinf, exp)
+from more_itertools import unique_everseen
```

Comment:
`more_itertools` is not a standard library package, and is currently not declared as a dependency of gwpy, however...